addons/sourcemod/scripting/SurfTimer.sp

```diff
@@ -612,6 +615,7 @@ public void OnClientDisconnect(int client)
 				g_fTimeinZone[client][zonegroup] += g_fTimeIncrement[client][zonegroup];
 			db_UpdatePRinfo(client, g_szSteamID[client], zonegroup);
 		}
+		delete g_aCheckpointsDifference[client];
```

Comment:
I think this line can be moved earlier inside this function. As example after `db_savePlayTime(client);`.